chalicelib/utils.py

```diff
@@ -27,14 +29,15 @@ class ManifestIO:
           - avoid writing to TSVs to disk altogether
     """
 
-    def __init__(self, data,
-                 url=None, workspace=None, namespace=None, auth=None):
+    def __init__(self, data, url=None, workspace=None,
+                 namespace=None, auth=None, statusCode=None):
```

Comment:
1. I don't think you should pass in the statusCode object; you should create it.
2. I think the Python naming convention would be status_code, not statusCode
3. I think the name should indicate that it holds multiple status codes, e.g., status_codes.

Reply:
it's part of the constructor now